### DISCUSSION TITLE
kafka cluster consumption accumulation, occupying high disk space.

### DISCUSSION THREAD

**Discussion by liucx1201** [apache/skywalking#11796] [Q&A]
Topic:
Skywalking segments-1
Skywalking segments-2

The occupancy above is high, with a backlog of billions of messages. Slowly rise。

The skwalking oat server has enabled two Docker nodes. ES has 3 nodes, and the utilization rate of ES IO is not high

Architecture: Skywalking agent ->Kafka ->Skywalking ->es

I don't know what caused it? Can you help me take a look?

What will affect Kafaka consumption, are there any emergency measures in place?

The current impact is that it is not possible to query data within 1 hour on the Skywalking UI

**Comment by liucx1201** [comment]
I'm not sure if it's related: https://github.com/apache/skywalking/issues/11795

**Comment by liucx1201** [comment]
![es](https://github.com/apache/skywalking/assets/40434832/8faf09c6-a147-4ad9-98a8-5cbce07d70d4)

**Comment by liucx1201** [comment]
![kafak-disk](https://github.com/apache/skywalking/assets/40434832/1e94b1b7-a007-4917-8169-d6a164e7c153)

**Comment by wu-sheng** [comment]
Personally, I don't have experience of setting the OAP to consume from Kafka. Maybe others know, but this is my use case.
